README.md

```diff
@@ -134,23 +134,13 @@ You can also build custom runtimes with libraries that your functions depends on
 
 ## Verify 
 
-To test that all is working, execute:
-```python
-from pywren_ibm_cloud import tests
-tests.run()
-```
-Notice that if you didn't set a local PyWren's config file, you need to provide it as a dictionary by the `config` parameter of `run()` method which uses the default local config file if this parameter is `None`. 
+To test that all is working, use the command:
 
-Alternatively, for debugging purposes, you can also run [tests.py](pywren_ibm_cloud/tests.py) directly which is located in the `pywren_ibm_cloud` folder with the arguments listed below.
+    python -m pywren_ibm_cloud.tests
```

Comment:
@omerb01 what is "-m" ?

Reply:
"-m" flag is using for operating python modules outside of python env, so after you install PyWren with this changes, you can run the test script from the terminal by this command easily.